<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Sebastian Trueg</author>
 <class>K3bLameEncoderSettingsWidget</class>
 <widget class="QWidget" name="K3bLameEncoderSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>291</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="m_mainTab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Quality Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QRadioButton" name="m_radioQualityLevel">
              <property name="text">
               <string>Preset:</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="m_labelQualityLevel">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>textLabel1</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QFrame" name="m_frameQualityLevel">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QGridLayout">
               <item row="1" column="1">
                <widget class="QLabel" name="textLabel4">
                 <property name="font">
                  <font>
                   <italic>true</italic>
                  </font>
                 </property>
                 <property name="text">
                  <string>high quality</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="wordWrap">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="textLabel3">
                 <property name="font">
                  <font>
                   <italic>true</italic>
                  </font>
                 </property>
                 <property name="text">
                  <string>small file</string>
                 </property>
                 <property name="wordWrap">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" colspan="2">
                <widget class="QSlider" name="m_sliderQuality">
                 <property name="maximum">
                  <number>9</number>
                 </property>
                 <property name="pageStep">
                  <number>3</number>
                 </property>
                 <property name="value">
                  <number>3</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QRadioButton" name="m_radioManual">
              <property name="text">
               <string>Manual settings:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="m_labelManualSettings">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>textLabel2</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>5</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="m_buttonManualSettings">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Change Settings...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Encoder Quality</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="KIntNumInput" name="m_spinEncoderQuality">
            <property name="toolTip">
             <string>Choose the noise shaping &amp; psycho acoustic algorithm.</string>
            </property>
            <property name="whatsThis">
             <string>&lt;p&gt;Bitrate is of course the main influence on quality. The higher the bitrate, the higher the quality. But for a given bitrate, we have a choice of algorithms to determine the best scalefactors and huffman encoding (noise shaping).
&lt;p&gt;The quality increases from 0 to 9 while the encoding speed drops.
&lt;p&gt;9 uses the slowest &amp; best possible version of all algorithms.
&lt;p&gt;&lt;b&gt;7 is the recommended setting&lt;/b&gt; while 4 still produced reasonable quality at good speed.
&lt;p&gt;0 disables almost all algorithms including psy-model resulting in poor quality.
&lt;p&gt;&lt;b&gt;This setting has no influence on the size of the resulting file.&lt;/b&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="textLabel1">
              <property name="font">
               <font>
                <italic>true</italic>
               </font>
              </property>
              <property name="text">
               <string>fast encoding</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="textLabel2">
              <property name="font">
               <font>
                <italic>true</italic>
               </font>
              </property>
              <property name="text">
               <string>high quality</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="m_checkCopyright">
            <property name="toolTip">
             <string>Mark the encoded file as being copyrighted.</string>
            </property>
            <property name="text">
             <string>Mark copyrighted</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_checkOriginal">
            <property name="toolTip">
             <string>Mark the encoded file as being a copy.</string>
            </property>
            <property name="text">
             <string>Mark as original</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_checkISO">
            <property name="toolTip">
             <string>Enforce strict ISO compliance</string>
            </property>
            <property name="whatsThis">
             <string>&lt;p&gt;If this option is checked, LAME will enforce the 7680 bit limitation on total frame size.&lt;br&gt;
This results in many wasted bits for high bitrate encodings but will ensure strict ISO compatibility. This compatibility might be important for hardware players.</string>
            </property>
            <property name="text">
             <string>Strict ISO compliance</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_checkError">
            <property name="toolTip">
             <string>Turn on CRC error protection.</string>
            </property>
            <property name="whatsThis">
             <string>&lt;p&gt;If this option is checked, a cyclic redundancy check (CRC) code will be added to each frame, allowing transmission errors that could occur on the MP3 stream to be detected; however, it takes 16 bits per frame that would otherwise be used for encoding, thus slightly reducing the sound quality.</string>
            </property>
            <property name="text">
             <string>Error protection</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_radioManual</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_buttonManualSettings</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_radioQualityLevel</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_labelQualityLevel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_radioManual</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_labelManualSettings</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_radioQualityLevel</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_frameQualityLevel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
